mapproxy/config/loader.py

```diff
@@ -1578,8 +1604,12 @@ def tile_layers(self, grid_name_as_path=False):
         for cache_name in sources:
             for grid, extent, cache_source in self.context.caches[cache_name].caches():
 
-                if dimensions and not isinstance(cache_source.cache, DummyCache):
+
+                from mapproxy.cache.s3 import S3Cache
+
+                if dimensions and not isinstance(cache_source.cache, DummyCache) and not isinstance(cache_source.cache, S3Cache):
```

Comment:
Could you explain why S3 caches (and dummy caches) are treated apart?
I can't really figure out why it would be so.